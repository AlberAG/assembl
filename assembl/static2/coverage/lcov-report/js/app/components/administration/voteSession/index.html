<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for js/app/components/administration/voteSession</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../../../index.html">All files</a> js/app/components/administration/voteSession
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">24.52% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>76/310</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">33.93% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>38/112</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">14.84% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>23/155</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">25.89% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>73/282</span>
      </div>
    </div>
  </div>
  <div class='status-line low'></div>
<div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file low" data-value="customizeGaugeForm.jsx"><a href="customizeGaugeForm.jsx.html">customizeGaugeForm.jsx</a></td>
	<td data-value="12.5" class="pic low"><div class="chart"><div class="cover-fill" style="width: 12%;"></div><div class="cover-empty" style="width:88%;"></div></div></td>
	<td data-value="12.5" class="pct low">12.5%</td>
	<td data-value="56" class="abs low">7/56</td>
	<td data-value="14.29" class="pct low">14.29%</td>
	<td data-value="14" class="abs low">2/14</td>
	<td data-value="8.57" class="pct low">8.57%</td>
	<td data-value="35" class="abs low">3/35</td>
	<td data-value="15.56" class="pct low">15.56%</td>
	<td data-value="45" class="abs low">7/45</td>
	</tr>

<tr>
	<td class="file low" data-value="gaugeForm.jsx"><a href="gaugeForm.jsx.html">gaugeForm.jsx</a></td>
	<td data-value="14.55" class="pic low"><div class="chart"><div class="cover-fill" style="width: 14%;"></div><div class="cover-empty" style="width:86%;"></div></div></td>
	<td data-value="14.55" class="pct low">14.55%</td>
	<td data-value="55" class="abs low">8/55</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="16" class="abs medium">8/16</td>
	<td data-value="9.52" class="pct low">9.52%</td>
	<td data-value="21" class="abs low">2/21</td>
	<td data-value="14.81" class="pct low">14.81%</td>
	<td data-value="54" class="abs low">8/54</td>
	</tr>

<tr>
	<td class="file low" data-value="modulesSection.jsx"><a href="modulesSection.jsx.html">modulesSection.jsx</a></td>
	<td data-value="24.53" class="pic low"><div class="chart"><div class="cover-fill" style="width: 24%;"></div><div class="cover-empty" style="width:76%;"></div></div></td>
	<td data-value="24.53" class="pct low">24.53%</td>
	<td data-value="53" class="abs low">13/53</td>
	<td data-value="44.44" class="pct low">44.44%</td>
	<td data-value="18" class="abs low">8/18</td>
	<td data-value="22.22" class="pct low">22.22%</td>
	<td data-value="27" class="abs low">6/27</td>
	<td data-value="21.28" class="pct low">21.28%</td>
	<td data-value="47" class="abs low">10/47</td>
	</tr>

<tr>
	<td class="file medium" data-value="numberGaugeForm.jsx"><a href="numberGaugeForm.jsx.html">numberGaugeForm.jsx</a></td>
	<td data-value="50" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 50%;"></div><div class="cover-empty" style="width:50%;"></div></div></td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="4" class="abs medium">2/4</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="4" class="abs medium">2/4</td>
	<td data-value="33.33" class="pct low">33.33%</td>
	<td data-value="3" class="abs low">1/3</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="4" class="abs medium">2/4</td>
	</tr>

<tr>
	<td class="file medium" data-value="pageForm.jsx"><a href="pageForm.jsx.html">pageForm.jsx</a></td>
	<td data-value="54.55" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 54%;"></div><div class="cover-empty" style="width:46%;"></div></div></td>
	<td data-value="54.55" class="pct medium">54.55%</td>
	<td data-value="22" class="abs medium">12/22</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="11.11" class="pct low">11.11%</td>
	<td data-value="9" class="abs low">1/9</td>
	<td data-value="57.14" class="pct medium">57.14%</td>
	<td data-value="21" class="abs medium">12/21</td>
	</tr>

<tr>
	<td class="file medium" data-value="textGaugeForm.jsx"><a href="textGaugeForm.jsx.html">textGaugeForm.jsx</a></td>
	<td data-value="75" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 75%;"></div><div class="cover-empty" style="width:25%;"></div></div></td>
	<td data-value="75" class="pct medium">75%</td>
	<td data-value="4" class="abs medium">3/4</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="66.67" class="pct medium">66.67%</td>
	<td data-value="3" class="abs medium">2/3</td>
	<td data-value="75" class="pct medium">75%</td>
	<td data-value="4" class="abs medium">3/4</td>
	</tr>

<tr>
	<td class="file low" data-value="tokenCategoryForm.jsx"><a href="tokenCategoryForm.jsx.html">tokenCategoryForm.jsx</a></td>
	<td data-value="28.57" class="pic low"><div class="chart"><div class="cover-fill" style="width: 28%;"></div><div class="cover-empty" style="width:72%;"></div></div></td>
	<td data-value="28.57" class="pct low">28.57%</td>
	<td data-value="14" class="abs low">4/14</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="4" class="abs medium">2/4</td>
	<td data-value="14.29" class="pct low">14.29%</td>
	<td data-value="7" class="abs low">1/7</td>
	<td data-value="30.77" class="pct low">30.77%</td>
	<td data-value="13" class="abs low">4/13</td>
	</tr>

<tr>
	<td class="file low" data-value="tokensForm.jsx"><a href="tokensForm.jsx.html">tokensForm.jsx</a></td>
	<td data-value="25" class="pic low"><div class="chart"><div class="cover-fill" style="width: 25%;"></div><div class="cover-empty" style="width:75%;"></div></div></td>
	<td data-value="25" class="pct low">25%</td>
	<td data-value="24" class="abs low">6/24</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="8" class="abs medium">4/8</td>
	<td data-value="30" class="pct low">30%</td>
	<td data-value="10" class="abs low">3/10</td>
	<td data-value="26.09" class="pct low">26.09%</td>
	<td data-value="23" class="abs low">6/23</td>
	</tr>

<tr>
	<td class="file low" data-value="voteProposalForm.jsx"><a href="voteProposalForm.jsx.html">voteProposalForm.jsx</a></td>
	<td data-value="25" class="pic low"><div class="chart"><div class="cover-fill" style="width: 25%;"></div><div class="cover-empty" style="width:75%;"></div></div></td>
	<td data-value="25" class="pct low">25%</td>
	<td data-value="64" class="abs low">16/64</td>
	<td data-value="25" class="pct low">25%</td>
	<td data-value="48" class="abs low">12/48</td>
	<td data-value="6.25" class="pct low">6.25%</td>
	<td data-value="32" class="abs low">2/32</td>
	<td data-value="27.59" class="pct low">27.59%</td>
	<td data-value="58" class="abs low">16/58</td>
	</tr>

<tr>
	<td class="file low" data-value="voteProposalsSection.jsx"><a href="voteProposalsSection.jsx.html">voteProposalsSection.jsx</a></td>
	<td data-value="35.71" class="pic low"><div class="chart"><div class="cover-fill" style="width: 35%;"></div><div class="cover-empty" style="width:65%;"></div></div></td>
	<td data-value="35.71" class="pct low">35.71%</td>
	<td data-value="14" class="abs low">5/14</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="25" class="pct low">25%</td>
	<td data-value="8" class="abs low">2/8</td>
	<td data-value="38.46" class="pct low">38.46%</td>
	<td data-value="13" class="abs low">5/13</td>
	</tr>

</tbody>
</table>
</div><div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Fri Oct 26 2018 15:51:30 GMT+0200 (CEST)
</div>
</div>
<script src="../../../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../../../sorter.js"></script>
</body>
</html>
