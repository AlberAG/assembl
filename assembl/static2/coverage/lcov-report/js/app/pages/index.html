<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for js/app/pages</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../prettify.css" />
    <link rel="stylesheet" href="../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../index.html">All files</a> js/app/pages
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">35.58% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>201/565</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">29.13% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>97/333</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">28.11% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>52/185</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">36.59% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>191/522</span>
      </div>
    </div>
  </div>
  <div class='status-line low'></div>
<div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file medium" data-value="brightMirrorFiction.jsx"><a href="brightMirrorFiction.jsx.html">brightMirrorFiction.jsx</a></td>
	<td data-value="70" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 70%;"></div><div class="cover-empty" style="width:30%;"></div></div></td>
	<td data-value="70" class="pct medium">70%</td>
	<td data-value="60" class="abs medium">42/60</td>
	<td data-value="69.05" class="pct medium">69.05%</td>
	<td data-value="42" class="abs medium">29/42</td>
	<td data-value="37.5" class="pct low">37.5%</td>
	<td data-value="16" class="abs low">6/16</td>
	<td data-value="70.91" class="pct medium">70.91%</td>
	<td data-value="55" class="abs medium">39/55</td>
	</tr>

<tr>
	<td class="file high" data-value="community.jsx"><a href="community.jsx.html">community.jsx</a></td>
	<td data-value="100" class="pic high"><div class="chart"><div class="cover-fill cover-full" style="width: 100%;"></div><div class="cover-empty" style="width:0%;"></div></div></td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	</tr>

<tr>
	<td class="file low" data-value="idea.jsx"><a href="idea.jsx.html">idea.jsx</a></td>
	<td data-value="36.92" class="pic low"><div class="chart"><div class="cover-fill" style="width: 36%;"></div><div class="cover-empty" style="width:64%;"></div></div></td>
	<td data-value="36.92" class="pct low">36.92%</td>
	<td data-value="130" class="abs low">48/130</td>
	<td data-value="31.51" class="pct low">31.51%</td>
	<td data-value="73" class="abs low">23/73</td>
	<td data-value="37.93" class="pct low">37.93%</td>
	<td data-value="29" class="abs low">11/29</td>
	<td data-value="38.1" class="pct low">38.1%</td>
	<td data-value="126" class="abs low">48/126</td>
	</tr>

<tr>
	<td class="file medium" data-value="question.jsx"><a href="question.jsx.html">question.jsx</a></td>
	<td data-value="61.54" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 61%;"></div><div class="cover-empty" style="width:39%;"></div></div></td>
	<td data-value="61.54" class="pct medium">61.54%</td>
	<td data-value="13" class="abs medium">8/13</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="6" class="abs low">0/6</td>
	<td data-value="33.33" class="pct low">33.33%</td>
	<td data-value="3" class="abs low">1/3</td>
	<td data-value="66.67" class="pct medium">66.67%</td>
	<td data-value="12" class="abs medium">8/12</td>
	</tr>

<tr>
	<td class="file medium" data-value="requestPasswordChange.jsx"><a href="requestPasswordChange.jsx.html">requestPasswordChange.jsx</a></td>
	<td data-value="57.14" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 57%;"></div><div class="cover-empty" style="width:43%;"></div></div></td>
	<td data-value="57.14" class="pct medium">57.14%</td>
	<td data-value="7" class="abs medium">4/7</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="2" class="abs high">2/2</td>
	<td data-value="25" class="pct low">25%</td>
	<td data-value="4" class="abs low">1/4</td>
	<td data-value="80" class="pct high">80%</td>
	<td data-value="5" class="abs high">4/5</td>
	</tr>

<tr>
	<td class="file medium" data-value="syntheses.jsx"><a href="syntheses.jsx.html">syntheses.jsx</a></td>
	<td data-value="50" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 50%;"></div><div class="cover-empty" style="width:50%;"></div></div></td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="12" class="abs medium">6/12</td>
	<td data-value="37.5" class="pct low">37.5%</td>
	<td data-value="16" class="abs low">6/16</td>
	<td data-value="60" class="pct medium">60%</td>
	<td data-value="5" class="abs medium">3/5</td>
	<td data-value="54.55" class="pct medium">54.55%</td>
	<td data-value="11" class="abs medium">6/11</td>
	</tr>

<tr>
	<td class="file low" data-value="synthesis.jsx"><a href="synthesis.jsx.html">synthesis.jsx</a></td>
	<td data-value="27.66" class="pic low"><div class="chart"><div class="cover-fill" style="width: 27%;"></div><div class="cover-empty" style="width:73%;"></div></div></td>
	<td data-value="27.66" class="pct low">27.66%</td>
	<td data-value="47" class="abs low">13/47</td>
	<td data-value="9.09" class="pct low">9.09%</td>
	<td data-value="33" class="abs low">3/33</td>
	<td data-value="28.57" class="pct low">28.57%</td>
	<td data-value="14" class="abs low">4/14</td>
	<td data-value="28.26" class="pct low">28.26%</td>
	<td data-value="46" class="abs low">13/46</td>
	</tr>

<tr>
	<td class="file medium" data-value="voteSession.jsx"><a href="voteSession.jsx.html">voteSession.jsx</a></td>
	<td data-value="51.33" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 51%;"></div><div class="cover-empty" style="width:49%;"></div></div></td>
	<td data-value="51.33" class="pct medium">51.33%</td>
	<td data-value="113" class="abs medium">58/113</td>
	<td data-value="36" class="pct low">36%</td>
	<td data-value="75" class="abs low">27/75</td>
	<td data-value="44" class="pct low">44%</td>
	<td data-value="50" class="abs low">22/50</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="106" class="abs medium">53/106</td>
	</tr>

<tr>
	<td class="file low" data-value="voteSessionAdmin.jsx"><a href="voteSessionAdmin.jsx.html">voteSessionAdmin.jsx</a></td>
	<td data-value="11.54" class="pic low"><div class="chart"><div class="cover-fill" style="width: 11%;"></div><div class="cover-empty" style="width:89%;"></div></div></td>
	<td data-value="11.54" class="pct low">11.54%</td>
	<td data-value="182" class="abs low">21/182</td>
	<td data-value="8.14" class="pct low">8.14%</td>
	<td data-value="86" class="abs low">7/86</td>
	<td data-value="4.76" class="pct low">4.76%</td>
	<td data-value="63" class="abs low">3/63</td>
	<td data-value="11.88" class="pct low">11.88%</td>
	<td data-value="160" class="abs low">19/160</td>
	</tr>

</tbody>
</table>
</div><div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Fri Oct 26 2018 15:51:30 GMT+0200 (CEST)
</div>
</div>
<script src="../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../sorter.js"></script>
</body>
</html>
