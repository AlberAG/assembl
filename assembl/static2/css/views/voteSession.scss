.votesession-page {
  .header-content {
    display: flex;
    flex-direction: column;
    align-items: flex-start;
  }

  .vote-instructions {
    padding: 0 20px;
    font-size: $font-size-m;
    font-family: $font-family4;
    max-width: 780px;
    text-align: justify;
    line-height: 32px;
  }

  .content-section {
    padding: 0;
  }

  .available-tokens {
    margin: 20px 0 40px;

    .category-available-tokens {
      display: flex;
      padding: 30px 20px;
      max-width: 780px;

      .tokens {
        padding-left: 50px;
        padding-top: 22px;
      }
    }
  }

  .available-tokens-sticky {
    position: fixed;
    top: 60px;
    width: 100%;
    z-index: 100;
    display: flex;
    justify-content: space-around;
    box-shadow: 3px 4px 7px #bbb;
    padding: 0 20px 12px;
    border-top: 1px solid $grey1;

    .separator {
      display: none;
    }

    .category-available-tokens {
      display: flex;

      .tokens {
        padding-left: 30px;
        padding-top: 20px;
      }
    }
  }

  .extra-margin-top {
    margin-top: 400px;
  }

  .tokens > * {
    margin: 5px;
    width: 30px;
    height: 30px;
  }

  .theme-box {
    &:first-child {
      margin: -70px 0 0;
    }
  }

  .proposal.row {
    margin: 90px 10px 30px;

    .proposal-title {
      font-family: $font-family;
      margin-top: 0;
    }

    .text {
      text-align: justify;
      padding-right: 20px;
    }

    .proposal-vote-modules {
      padding-top: 25px;
    }

    .tokens-line {
      margin-top: 14px;
      margin-bottom: 30px;

      .tokens {
        margin-left: -5px;
      }
    }

    .tokens > button {
      border: none;
      padding: 0;
      background: none;

      &:focus {
        outline: none;
      }

      &:active {
        box-shadow: none;
      }
    }

    .tokens > .btn-default:hover {
      background-color: transparent;
    }
  }

  .gauge-vote-for-proposal {
    margin-top: 40px;

    p {
      margin-bottom: 40px;
      margin-left: -5px;
    }
  }

  .form-actions {
    margin-top: 50px;
    margin-bottom: 100px;
  }

  .link-button {
    border: none;
    font-size: $font-size-xs;
    color: $link-color;
    text-decoration: none;
    background-color: $grey0;
    padding: 15px;

    &:hover {
      color: $link-hover-color;
      text-decoration: none;
    }

    &:focus {
      outline: none;
    }

    &:active {
      box-shadow: none;
    }

    [class^='assembl-icon-']:before,
    [class*=' assembl-icon-']:before {
      font-size: $font-size-xxs;
      margin-left: 5px;
    }
  }

  .link-button:focus,
  .link-button:active,
  .link-button:hover {
    background-color: transparent;
  }

  .expand-row {
    margin: 0 40px;
  }

  .votes-in-progress {
    margin: 0 40px;
    padding: 0 15px 30px;

    .box {
      margin-top: 30px;
      height: 160px;
    }

    .participants-count {
      .assembl-icon-profil:before {
        font-size: 40px;
        margin-bottom: 12px;
      }
    }

    .doughnut-container {
      margin: 0 auto;
      height: 80px;
      width: 80px;
      padding: 10px 0 0;

      .doughnut {
        .circle {
          stroke-width: 22px;
        }
      }
    }
  }

  .expand-result {
    padding-bottom: 40px;
  }

  .gauge-box {
    padding: 30px 0 0;
  }

  .separator {
    border-bottom: 1px $btn-dark-border-style $grey1;
    max-width: 745px;
    margin-left: 18px;
  }
}

.token-tooltip {
  border-radius: 4px;

  .tooltip-inner {
    background: none;
  }
}

@media screen and (min-width: 768px) {
  .votesession-page {
    .proposal.row {
      padding: 31px 15px 15px 15px;
    }

    .participants-count {
      padding: 40px 0 0;
    }

    .tokens-box {
      padding: 30px 0 0;
    }
  }
}

@media screen and (max-width: 767px) {
  .votesession-page {
    .available-tokens-sticky {
      display: none;
    }

    .container {
      padding: 0;
      margin: 0;
    }

    .proposal.row {
      padding: 15px 0 0;
    }

    .participants-count {
      padding: 32px 0 0;
    }

    .tokens-box {
      padding: 15px 0 0;
    }
  }
}
