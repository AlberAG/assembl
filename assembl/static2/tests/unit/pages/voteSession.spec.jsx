import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import { DumbVoteSession } from '../../../js/app/pages/voteSession';

describe('VoteSession component', () => {
  it('should match VoteSession snapshot', () => {
    const voteForProposalSpy = jest.fn();
    const props = {
      title: 'Phase de vote à la majorité et estimation multicritère',
      subTitle: 'Maintenant il faut voter.',
      headerImageUrl: 'foo.jpg',
      instructionsSectionTitle: 'Instructions de la phase de vote',
      instructionsSectionContent: 'Elisez les meilleurs propositions, vous avez 8 jetons pour et 3 jetons contre',
      modules: [],
      propositionsSectionTitle: 'Vote sur 10 propositions',
      proposals: [
        { id: 'foo', title: 'Foo', description: 'You can\'t hack the alarm without connecting the primary AGP microchip!' }
      ],
      voteForProposal: voteForProposalSpy
    };
    const shallowRenderer = new ShallowRenderer();
    shallowRenderer.render(<DumbVoteSession {...props} />);
    const result = shallowRenderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});