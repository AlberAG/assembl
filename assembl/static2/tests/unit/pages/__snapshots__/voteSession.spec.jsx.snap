// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VoteSession component should match VoteSession snapshot 1`] = `
<div
  className="votesession-page"
>
  <Connect(Header)
    additionalHeaderClasses="left"
    imgUrl="foo.jpg"
    subtitle="Maintenant il faut voter."
    title="Phase de vote à la majorité et estimation multicritère"
  />
  <Grid
    bsClass="container"
    className="background-light"
    componentClass="div"
    fluid={true}
  >
    <Section
      className="themes-section"
      containerAdditionalClassNames={null}
      displayIndex={false}
      index={1}
      indexGenerator={[Function]}
      parents={Array []}
      title="Instructions de la phase de vote"
      translate={false}
    >
      <Row
        bsClass="row"
        componentClass="div"
      >
        <Col
          bsClass="col"
          className="no-padding"
          componentClass="div"
          md={8}
          mdOffset={3}
          sm={10}
          smOffset={1}
        >
          <div
            className="vote-instructions"
            dangerouslySetInnerHTML={
              Object {
                "__html": "Elisez les meilleurs propositions, vous avez 8 jetons pour et 3 jetons contre",
              }
            }
          />
          <div>
            <AvailableTokens
              remainingTokensByCategory={
                Immutable.Map {
                  negative: 10,
                  positive: 8,
                }
              }
              sticky={false}
              tokenCategories={
                Array [
                  Object {
                    "color": "#117711",
                    "id": "negative",
                    "title": "Negative",
                    "totalNumber": 10,
                  },
                  Object {
                    "color": "#881111",
                    "id": "positive",
                    "title": "Positive",
                    "totalNumber": 8,
                  },
                ]
              }
            />
          </div>
        </Col>
      </Row>
    </Section>
  </Grid>
  <Grid
    bsClass="container"
    className="background-grey"
    componentClass="div"
    fluid={true}
  >
    <Section
      className={null}
      containerAdditionalClassNames={Array []}
      displayIndex={false}
      index={1}
      indexGenerator={[Function]}
      parents={Array []}
      title="Vote sur 10 propositions"
      translate={false}
    >
      <Row
        bsClass="row"
        componentClass="div"
      >
        <Col
          bsClass="col"
          componentClass="div"
          md={10}
          mdOffset={1}
          sm={10}
          smOffset={1}
        >
          <Proposals
            proposals={
              Array [
                Object {
                  "description": "You can't hack the alarm without connecting the primary AGP microchip!",
                  "id": "proposal-1",
                  "modules": Array [
                    Object {
                      "exclusiveCategories": true,
                      "id": "token-vote-module",
                      "instructions": "If we generate the feed, we can get to the AGP driver through the primary ADP interface!",
                      "tokenCategories": Array [
                        Object {
                          "color": "#226622",
                          "id": "positive",
                          "title": "Positive",
                          "totalNumber": 12,
                        },
                        Object {
                          "color": "#771122",
                          "id": "negative",
                          "title": "Negative",
                          "totalNumber": 10,
                        },
                      ],
                      "voteType": "token_vote_specification",
                    },
                  ],
                  "title": "First proposal",
                  "voteResults": Object {
                    "numParticipants": 1,
                  },
                },
                Object {
                  "description": "I'll input the multi-byte SAS monitor, that should bandwidth the USB microchip!",
                  "id": "proposal-2",
                  "modules": Array [
                    Object {
                      "exclusiveCategories": true,
                      "id": "token-vote-module",
                      "instructions": "If we generate the feed, we can get to the AGP driver through the primary ADP interface!",
                      "tokenCategories": Array [
                        Object {
                          "color": "#226622",
                          "id": "positive",
                          "title": "Positive",
                          "totalNumber": 12,
                        },
                        Object {
                          "color": "#771122",
                          "id": "negative",
                          "title": "Negative",
                          "totalNumber": 10,
                        },
                      ],
                      "voteType": "token_vote_specification",
                    },
                    Object {
                      "id": "text-gauge-module",
                      "instructions": "I'll generate the mobile SMS bus, that should microchip the TCP matrix!",
                      "voteType": "gauge_vote_specification",
                    },
                  ],
                  "title": "Second proposal",
                  "voteResults": Object {
                    "numParticipants": 2,
                  },
                },
              ]
            }
            remainingTokensByCategory={
              Immutable.Map {
                negative: 10,
                positive: 8,
              }
            }
            userTokenVotes={
              Immutable.Map {
              }
            }
            voteForProposal={[Function]}
            voteForProposalGauge={[Function]}
          />
        </Col>
      </Row>
      <Row
        bsClass="row"
        className="form-actions center"
        componentClass="div"
      >
        <Col
          bsClass="col"
          componentClass="div"
          md={10}
          mdOffset={1}
          sm={10}
          smOffset={1}
        />
      </Row>
    </Section>
  </Grid>
</div>
`;

exports[`VoteSession component should match VoteSession snapshot when vote session is not configured 1`] = `
<MessagePage
  text="Text"
  title="Title"
/>
`;

exports[`VoteSession component should match VoteSession snapshot with empty modules and proposals 1`] = `
<div
  className="votesession-page"
>
  <Connect(Header)
    additionalHeaderClasses="left"
    imgUrl="foo.jpg"
    subtitle="Maintenant il faut voter."
    title="Phase de vote à la majorité et estimation multicritère"
  />
  <Grid
    bsClass="container"
    className="background-light"
    componentClass="div"
    fluid={true}
  >
    <Section
      className="themes-section"
      containerAdditionalClassNames={null}
      displayIndex={false}
      index={1}
      indexGenerator={[Function]}
      parents={Array []}
      title="Instructions de la phase de vote"
      translate={false}
    >
      <Row
        bsClass="row"
        componentClass="div"
      >
        <Col
          bsClass="col"
          className="no-padding"
          componentClass="div"
          md={8}
          mdOffset={3}
          sm={10}
          smOffset={1}
        >
          <div
            className="vote-instructions"
            dangerouslySetInnerHTML={
              Object {
                "__html": "",
              }
            }
          />
        </Col>
      </Row>
    </Section>
  </Grid>
  <Grid
    bsClass="container"
    className="background-grey"
    componentClass="div"
    fluid={true}
  >
    <Section
      className={null}
      containerAdditionalClassNames={Array []}
      displayIndex={false}
      index={1}
      indexGenerator={[Function]}
      parents={Array []}
      title="Vote sur 0 propositions"
      translate={false}
    >
      <Row
        bsClass="row"
        componentClass="div"
      >
        <Col
          bsClass="col"
          componentClass="div"
          md={10}
          mdOffset={1}
          sm={10}
          smOffset={1}
        >
          <Proposals
            proposals={Array []}
            remainingTokensByCategory={
              Immutable.Map {
              }
            }
            userTokenVotes={
              Immutable.Map {
              }
            }
            voteForProposal={[Function]}
            voteForProposalGauge={[Function]}
          />
        </Col>
      </Row>
      <Row
        bsClass="row"
        className="form-actions center"
        componentClass="div"
      >
        <Col
          bsClass="col"
          componentClass="div"
          md={10}
          mdOffset={1}
          sm={10}
          smOffset={1}
        />
      </Row>
    </Section>
  </Grid>
</div>
`;
